devito/types/basic.py

```diff
@@ -426,6 +431,13 @@ def is_const(self):
         """
         return self._is_const
 
+    @property
+    def nonnegative(self):
+        """
+        Alias for is_nonnegative
```

Comment:
why the need? sympy convention is "is_..."

Reply:
Because whilst the attribute is `is_nonnegative`, the argument that sets it is just `nonnegative`, so it's needed for rebuilding

Reply:
Not sure I follow why it's needed and why `is_nonnegative` is problematic

Reply:
`is_nonnegative` isn't problematic, but it's set with `Symbol(name='x', nonnegative=True)`. Hence rebuilding requires the alias